gazelle/testdata/folder_rules/my_module/BUILD.out

```diff
@@ -0,0 +1,28 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_project")
+load("@npm//jest:index.bzl", "jest_test")
+
+# gazelle:js_folder_as_rule
+# gazelle:js_test_shard_count 2
+# gazelle:js_test_size large
+
+jest_test(
+    name = "my_module_test",
+    size = "large",
+    srcs = [
+        "nested_module/a.test.ts",
+        "nested_module/b.test.ts",
+    ],
+    shard_count = 2,
+)
+
+ts_project(
+    name = "my_module",
+    srcs = [
+        "index.ts",
+        "module_file.ts",
+        "nested_module/index.ts",
+        "nested_module/module_file.ts",
+        "some_other_file.ts",
```

Comment:
this might become too long, can we use `glob` here

Reply:
So srcs is used to map imports to rules. It might make sense to try cleaning it up with `glob` later.